src/lib/analytics.js

```diff
@@ -1,7 +1,7 @@
 import posthog from "posthog-js";
 
 function initiatePostHog () {
-    posthog.init(process.env.POSTHOG_ID, { api_host: 'https://app.posthog.com' });
+    posthog.init(process.env.VITE_POSTHOG_ID, { api_host: 'https://app.posthog.com' });
```

Comment:
Per this [PR](https://github.com/open-sauced/open-sauced/pull/1442/files#diff-a96ec4e9261995cfb834805e073040556daa2d946f40771a73c228f2f5589004R4), it should be `import.meta.VITE_POSTHOG_ID`

Reply:
Nvm, but the resolution is allow listing the urls in posthog and exposing the id